service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/handlers/C2UploadedEventHandlerEmailTemplateTest.java

```diff
@@ -0,0 +1,96 @@
+import uk.gov.hmcts.reform.fpl.request.RequestData;
+import uk.gov.hmcts.reform.fpl.service.email.NotificationService;
+import uk.gov.hmcts.reform.fpl.service.email.content.C2UploadedEmailContentProvider;
+import uk.gov.hmcts.reform.fpl.testingsupport.email.EmailTemplateTest;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import static org.mockito.BDDMockito.given;
+
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.CASE_TYPE;
+import static uk.gov.hmcts.reform.fpl.CaseDefinitionConstants.JURISDICTION;
+import static uk.gov.hmcts.reform.fpl.enums.UserRole.LOCAL_AUTHORITY;
+import static uk.gov.hmcts.reform.fpl.handlers.NotificationEventHandlerTestData.AUTH_TOKEN;
+import static uk.gov.hmcts.reform.fpl.testingsupport.email.EmailContent.emailContent;
+import static uk.gov.hmcts.reform.fpl.testingsupport.email.SendEmailResponseAssert.assertThat;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.caseData;
+
+@SpringBootTest(classes = {
+    C2UploadedEventHandler.class,
+    NotificationService.class,
+    ObjectMapper.class
+})
+
+public class C2UploadedEventHandlerEmailTemplateTest extends EmailTemplateTest {
+    private final C2DocumentBundle c2DocumentBundle = C2DocumentBundle.builder().build();
+    private final String respondentLastName = "Smith";
+    private final String callout = "Smith, SACCCCCCCC5676576567";
+    private final String caseUrl = "null/case/" + JURISDICTION + "/" + CASE_TYPE + "/12345#C2Tab";
+
+    @Autowired
+    private C2UploadedEventHandler underTest;
+
+    @MockBean
+    private IdamClient idamClient;
+
+    @MockBean
+    private RequestData requestData;
+
+    @MockBean
+    private C2UploadedEmailContentProvider c2UploadedEmailContentProvider;
+
+    @Test
+    void notifyAdmin() {
+        CaseData caseData = caseData();
+
+        given(requestData.authorisation()).willReturn(AUTH_TOKEN);
+
+        given(idamClient.getUserInfo(AUTH_TOKEN)).willReturn(
+            UserInfo.builder().sub("hmcts-non-admin@test.com").roles(LOCAL_AUTHORITY.getRoles()).build());
+
+        given(c2UploadedEmailContentProvider.getNotifyData(caseData,
+            c2DocumentBundle.getDocument()))
+            .willReturn(
+                C2UploadedTemplate.builder()
+                    .callout(callout)
+                    .respondentLastName(respondentLastName)
+                    .caseUrl(caseUrl)
+                    .documentLink(DOCUMENT_LINK)
+                    .build()
+            );
+
+        underTest.notifyAdmin(new C2UploadedEvent(caseData, c2DocumentBundle));
+
+        assertThat(response())
+            .hasSubject("C2 application received, " + respondentLastName)
+            .hasBody(emailContent()
+                .line("A C2 application has been received for the case:")
+                .line()
+                .line("^ " + callout)//callout
```

Comment:
You could have a specific method called `callout()` that could be chain called here and would prefix the parameter with `"^ "`, other tests will need it and it means that you just have to focus on what the text will be instead of the format, same for the header and list below. You could have a h1 method which prefixes whatever is passed with `#` and a list method that takes in a var args of string and prefixes each one with `* `.